.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,41 @@
+---
+name: 'Bug Report'
+about: 'Report a bug, something does not work as it supposed to'
+title: '[bug] SHORT DESCRIPTION'
+labels: bug
+---
+
+<!--
+  Please don't forget to update the issue title.
+  Include all applicable information to help us reproduce your problem.
+
+  To help us debug your issue please explain:
+-->
+
+### Package and Environment Details (include every applicable attribute)
```

Comment:
All this makes no sense at all to report a bug.

Reply:
I've removed (updated) this section to obtain only environment information